COMMITTERS.md

```diff
@@ -2,28 +2,32 @@ Active Committers
 =================
 
 * Benjy Weinberger
-* Chris Heisterkamp
 * Daniel McClanahan
 * Daniel Wagner-Hall
-* Dorothy Ordogh
+* Eric Arellano
+* Greg Shuflin
+* Henry Fuller
 * Ity Kaul
 * John Sirois
-* Kris Wilson
-* Larry Hosken
-* Mateo Rodriguez
 * Nora Howard
+* Pierre Chevalier
 * Stu Hood
 * Yi Cheng
 
 Emeritus
```

Comment:
Given the below:

> The decision to place a committer on emeritus status is either by request, or by majority vote of the committers.  If you have not participated in code reviews or contributed a patch to pants within the past three months an automatic recommendation for emeritus status may be put forward to the committers.

...should maybe start a committers list thread about doing a batch edit to move these folks over?